doc/source/dev/contributor/quickstart_ubuntu.rst

```diff
@@ -31,7 +32,9 @@ Building SciPy
 
 #. Download, install, and test the latest release of the `Anaconda Distribution of Python`_. In addition to the latest version of Python 3, the Anaconda distribution includes dozens of the most popular Python packages for scientific computing, the Spyder integrated development environment (IDE), the ``conda`` package manager, and tools for managing virtual environments.
 
-   In Ubuntu, the Anaconda download comes as a ``.sh`` file. `In a terminal, navigate <https://help.ubuntu.com/community/UsingTheTerminal>`_ to the directory in which the ``.sh`` file is located and enter ``./file.sh``, where ``file.sh`` is to be replaced with the full name of the Anaconda download. This starts the installation process; from there, simply follow the prompts.
+ `In a terminal, navigate <https://help.ubuntu.com/community/UsingTheTerminal>`_ to the location in which you'd like to install Anaconda. You can download the file using the terminal command ``curl -O URL_OF_FILE``, where ``URL_OF_FILE`` is to be replaced by the URL of the Anaconda installer ``.sh`` file found at the Anaconda Distribution website. When the download is complete, enter ``chmod +x file.sh``, where ``file.sh`` is to be replaced with the full name of the downloaded file. This ensures that the file's permissions allow it to be executed. Next, enter ``./file.sh``, where ``file.sh`` is again to be replaced with the full name of the downloaded file. This starts the installation process. From there, simply follow the prompts, including the "Next Steps" at the end after the installer finishes.
```

Comment:
You can also just run `bash Anaconda3-2019.03-Linux-x86_64.sh` instead of making it executable?  That's what it says on https://docs.anaconda.com/anaconda/install/linux/

Reply:
Good idea.